features/keychain/module/crypto/secp256k1.js

```diff
@@ -8,6 +8,18 @@ import { tweakPrivateKey } from './tweak'
 const isValidEcOptions = (ecOptions) =>
   !ecOptions || Object.keys(ecOptions).every((key) => ['canonical'].includes(key))
 
+const encodeSignature = ({ signature, enc }) => {
+  if (enc === 'der') return Buffer.from(signature.toDER())
+
+  const sig = { ...signature }
+
+  if (enc === 'raw') return sig
+
+  const r = Buffer.from(sig.r.toArray('be', 32))
+  const s = Buffer.from(sig.s.toArray('be', 32))
+  return Buffer.concat([r, s, Buffer.from([sig.recoveryParam])])
```

Comment:
should we add a unit test for this one?

Reply:
Will do. I was waiting to see if the idea is accepted.

Reply:
https://github.com/ExodusMovement/exodus-oss/pull/107/commits/9a109b470acc6831743dd9a78fbe096752baac14